crates/evm/src/tests/queries/mod.rs

```diff
@@ -368,12 +410,13 @@ pub fn init_evm_with_caller_contract() -> (Evm<C>, WorkingSet<C>, TestSigner) {
         .unwrap();
     }
 
-    evm.end_soft_confirmation_hook(&mut working_set);
+    evm.end_soft_confirmation_hook(&soft_confirmation_info, &mut working_set);
```

Comment:
Can we pass `soft_confirmation_info` instead of `&soft_confirmation_info` to end_ functions?

Reply:
yeah i guess it works

Reply:
to runtime functions we can, to module functions we can unfortunately